db/model/botData.js

```diff
@@ -0,0 +1,56 @@
+/**
+ * Copyright (c) 2017, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or
+ * https://opensource.org/licenses/BSD-3-Clause
+ */
+
+/**
+ * db/model/botData.js
+ *
+ * Bot webapps will have data that refocus will want to retrieve
+ * or store. This table will define the data information to be
+ * used manipulate the bots data.
+ */
```

Comment:
What are examples of such data? I'm having a hard time to find examples that would tie to the Bot model. If they exist - maybe should be part of the Bot model available as fields? 
All examples I can think of have a relationship with a Room model.